crates/forge/src/test_execution_syscall_handler.rs

```diff
@@ -645,18 +657,19 @@ fn handle_deploy_result(
         Err(CheatcodeError::Unrecoverable(err)) => Err(err),
     }
 }
-
-fn deserialize_cheat_target(inputs: &[Felt252]) -> CheatTarget {
+//
```

Comment:
I guess some comment about the function was meant to be here

Reply:
Right 😅